src/python/pants/core/register.py

```diff
@@ -33,22 +33,22 @@ def rules():
         *package.rules(),
```

Comment:
Thanks for fixing the ordering :)

Reply:
Yeah, it was hurting my eyes, and `C-space C-s ] C-p M-x so-li enter` is a mere 2.5 second fix :P